composer.json

```diff
@@ -50,7 +50,7 @@
         "mezzio/mezzio-helpers": "^5.3"
     },
     "require-dev": {
-        "composer/composer": "^1.5",
+        "composer/composer": "^1.5 || ^2.0@dev",
```

Comment:
While this gets it _ready_ for 2.0, it's pinned to a dev version. I'd argue we should do the following:

```
"^1.5 || ~2.0.0@dev || ^2.0"
```

This will allow users of a stable 2.0 to use that once it is released.

Reply:
I agree with that, but... 
I am not sure, isn't `^2.0@dev` the same as `~2.0.0@dev || ^2.0`?

That way it was provided in Ocramius package by Jordi, see:
https://github.com/Ocramius/PackageVersions/pull/133

Reply:
I've updated to use `"^1.5 || ~2.0.0@dev || ^2.0"`